server/plugin.go

```diff
@@ -4,91 +4,143 @@
 package main
 
 import (
-	"crypto/subtle"
+	"crypto/rsa"
 	"fmt"
-	"net/http"
+	"path/filepath"
 	"sync"
+	"text/template"
+
+	"github.com/pkg/errors"
 
-	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mattermost-server/plugin"
 )
 
+const (
+	JIRA_USERNAME = "Jira Plugin"
```

Comment:
OT: I would like to see the use of CamelCase instead of ALL_CAPS for plugins even though it would be inconsistent with the server code.